recipe/meta.yaml

```diff
@@ -1,17 +1,15 @@
-{% set version = "1.3" %}
+{% set version = "2.1.0" %}
 
 package:
-  name: flit
+  name: flit-split
```

Comment:
It's my understanding that this will create a `flit-split` metapackage. Is that what you intended here?

Reply:
Not in this case. The name is ignored in this case as it isn't mentioned in `outputs`.

A better way of working around this conda-build "feature" is to use the name of the first output (`flit-core` in this case).